examples/CURL/CMakeLists.txt

```diff
@@ -5,15 +5,21 @@ cmake_minimum_required(VERSION 3.0)
 
 # Emulate HunterGate:
 # * https://github.com/hunter-packages/gate
+set(TESTING_CONFIG_OPT FILEPATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Hunter/config.cmake) # Force OpenSSL 1.1.0a
+
 include("../common.cmake")
 
 project(download-curl)
 
+hunter_add_package(OpenSSL)
```

Comment:
This package will be installed automatically so there is no need to have this line. If you want to test such configuration please create separate example.